tests/sources/fixtures/google_drive/connector.json

```diff
@@ -15,6 +15,63 @@
       "order": 1,
       "ui_restrictions": []
+    },
+    "google_workspace_admin_email": {
+      "depends_on": [
+        {
+          "field": "use_document_level_security",
+          "value": true
+        }
+      ],
+      "display": "text",
+      "tooltip": "In order to use Document Level Security you need to enable Google Workspace domain-wide delegation of authority for your service account. A service account with delegated authority can impersonate admin user with sufficient permissions to fetch all users and their corresponding permissions.",
+      "default_value": null,
+      "label": "Google Workspace admin email",
+      "sensitive": false,
+      "type": "str",
+      "required": true,
+      "options": [],
+      "validations": [],
+      "value": "admin@your-organization.com",
+      "order": 3,
+      "ui_restrictions": []
+    },
+    "use_document_level_security": {
+      "depends_on": [],
+      "display": "toggle",
+      "tooltip": "Document level security ensures identities and permissions set in Google Drive are maintained in Elasticsearch. This enables you to restrict and personalize read-access users and groups have to documents in this index. Access control syncs ensure this metadata is kept up to date in your Elasticsearch documents.",
+      "default_value": null,
+      "label": "Enable document level security",
+      "sensitive": false,
+      "type": "bool",
+      "required": true,
+      "options": [],
+      "validations": [],
+      "value": false,
+      "order": 2,
+      "ui_restrictions": []
+    },
+    "max_concurrency": {
+      "depends_on": [],
+      "display": "numeric",
+      "tooltip": null,
```

Comment:
nit: Should we add a tooltip elaborating a little bit, which type of open connections?

Reply:
I added tooltip in the actual connector implementation, it was left as `null` in ftests for some reason